interfaces/matlab_experimental/Reactor/Valve.m

```diff
@@ -33,14 +33,10 @@
 
     methods
 
-        function v = Valve(upstream, downstream, name)
+        function v = Valve(varargin)
             % Constructor
 
-            if nargin < 3
-                name = '(none)';
-            end
-
-            v@FlowDevice('Valve', upstream, downstream, name);
+            v@FlowDevice('Valve', varargin{:});
```

Comment:
Like the other place where @ischoegl commented, I think this was better without `varargin`.